types/mime-db/index.d.ts

```diff
@@ -1,21 +1,33 @@
 // Type definitions for mime-db 1.27
 // Project: https://github.com/jshttp/mime-db
 // Definitions by: AJP <https://github.com/AJamesPhillips>
+//                 Linus Unnebäck <https://github.com/LinusU>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.2
+// TypeScript Version: 3.4
```

Comment:
This change seems redundant.

Reply:
To clarify, you should only bump it as high as you _need_ to make that required changes, if you have to mass bump other packages that's usually a bad sign.